client/src/components/FoodSeeker/SearchResults/ResultsMap/ResultsMap.js

```diff
@@ -162,13 +162,18 @@ const ResultsMap = (
       mapStyle={MAPBOX_STYLE}
       {...viewport}
       onViewportChange={setViewport}
-      onLoad={onLoad}
+      onLoad={(event) => {
+        event.target.removeControl(event.target.attributionControl);
```

Comment:
Not sure if removing attribution is part of this issue.

Reply:
Sorry, I did not see the map class in style.js completely earlier. Will do more analysis and update my review if required, but resolving this for now.